package/kedro_viz/server.py

```diff
@@ -452,10 +452,25 @@ def format_pipeline_data(
                 edges_list.append(edge)
             namespace_tags[namespace].update(node.tags)
             node_dependencies[task_id].add(namespace_id)
-
+    # Parameters and data
     for namespace, tag_names in sorted(namespace_tags.items()):
         is_param = bool("param" in namespace.lower())
         node_id = _hash(namespace)
+        if not is_param:
+
+            if KEDRO_VERSION.match(">=0.16.0"):
+                try:
+                    # pylint: disable=protected-access
+                    dataset = _CATALOG._get_dataset(namespace)
+                    _NODES[node_id] = {"type": "data", "obj": dataset}
+                except DataSetNotFoundError:
+                    pass
+            else:
+                dataset = _CATALOG._data_sets.get(namespace)
```

Comment:
can you use `_CATALOG.datasets`?

Reply:
Unfortunately, it doesn't exist in Kedro 0.14.0... https://github.com/quantumblacklabs/private-kedro/blob/release/0.14.0/kedro/io/data_catalog.py#L112

Reply:
We should drop the support for such old versions

Reply:
Yeah, we have a ticket for dropping Kedro 0.14 in KED-1696, and will simplify the backend.